docs/openapi/components/schemas/common/DCSATransportDocument.yml

```diff
@@ -186,5 +186,169 @@ required:
+            "addressLocality": "Antwerpen",
+            "postalCode": "BE-2100AN",
+            "addressCountry": "BE"
+          },
+          "phoneNumber": "+32-3-201.98.10"
+        },
+        "cargoItems": [
+          {
+            "type": "CargoItem",
+            "cargoLineItems": [
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators",
+                "shippingMarks": "ByAcre Age is the new Black"
+              },
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators Organizer Bags",
+                "shippingMarks": "ByAcre Age is the new Black"
+              },
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators Backrest",
+                "shippingMarks": "ByAcre Age is the new Black"
+              },
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators Cane Holder Bracket",
+                "shippingMarks": "ByAcre Age is the new Black"
+              },
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators Cardboard Box",
+                "shippingMarks": "ByAcre Age is the new Black"
+              }
+            ],
+            "weight": 1671.9,
+            "weightUnit": "KGM",
+            "volume": 27.23,
+            "volumeUnit": "CBM",
+            "numberOfPackages": 245
+          }
+        ],
+        "utilizedTransportEquipments": {
+          "type": "TransportEquipment",
+          "equipmentReference": "YMLU3380910",
+          "ISOEquipmentCode": "20G1",
+          "seals": [
+            {
+              "type": "Seal",
+              "sealNumber": "YMAI715692"
+            }
+          ]
+        }
+      },
+      "transports": {
+        "type": "Transport",
+        "loadLocation": {
+          "type": "Place",
+          "address": {
+            "type": "PostalAddress",
+            "addressRegion": "Xiamen",
+            "addressCountry": "CN"
+          }
+        },
+        "dischargeLocation": {
+          "type": "Place",
+          "address": {
+            "type": "PostalAddress",
+            "addressRegion": "Antwerp",
+            "addressCountry": "BE"
+          }
+        },
+        "modeOfTransport": "Vessel",
+        "carrier": {
+          "type": "Organization",
+          "name": "MULTI CONTAINER LINE",
+          "address": {
+            "type": "PostalAddress",
+            "organizationName": "MCL Multi Container Line LTD.",
+            "streetAddress": "Rm. 3501, 35/F Manhatten Place, 23 Wang Tai Road",
+            "addressLocality": "Kowloon Bay",
+            "addressRegion": "Hong Kong",
+            "addressCountry": "Hong Kong SAR"
+          }
+        },
+        "vesselNumber": "HMM Algeciras",
+        "voyageNumber": "V.0004W"
+      }
+    },
+    "@context": [
```

Comment:
I kinda don't like the sort order of these fields, prefer this at the top

Reply:
For sure, completely agree.